src/app/search/helpers.spec.ts

```diff
@@ -0,0 +1,41 @@
+/*
+ *    Copyright 2024 OICR, UCSC
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+import { parseTerms } from './helpers';
+
+describe('helpers', () => {
```

Comment:
Suggest a test for string with a single double quote, e.g., the closing quote is missing. Or maybe there is such a test already and I'm not seeing it. I'm not even sure what the behavior should be.

And, super-duper edge case, which I'm OK ignoring, can double quotes be escaped, e.g., what if I wanted to search for a double quote or a phrase I remembered had double quotes, e.g., I want to search for `"Santa Cruz"`, not `Santa Cruz`?

Reply:
I added some tests for the missing quote.  I'm also not sure what the correct behavior is, either, and as long as it's reasonable, it really doesn't matter.  So basically, the new tests parse the single double-quoted strings, confirm that they don't make the parser throw.

Re  "searching for quotes", the parser can handle escapes, but we'd need to adjust the query code to do an exact string search, and perhaps the indices.  IMHO, at this juncture, better to focus our energy on other things...